samples/Java/Basic/ReceiveUsingSequenceNumber/README.md

```diff
@@ -0,0 +1,30 @@
+# Receive events from Azure Event Hubs using Java
+
+This sample shows how to receive events from a particular Event Hub partition based on a sequence number. This is a lower-level API gesture that most applications will not use, but rather lean on the Event Processor Host to manage partition ownership and check-pointing. The [Event Processor Sample](../EventProcessorSample) shows this higher level functionality.
```

Comment:
please see if we can get 1 critical info straight out to customer:

- Creating receiver based on sequence# is expensive. To get best performance - create one based on Offset. Creating by seq# - service internally tries to resolve this to Offset.